### ISSUE TITLE
Custom abbreviation

### ISSUE THREAD

**Issue by gchegr** [isaaclyman/novel-word-count-obsidian#23]
**Problem**

I want to use the right alignment and no abbreviation at all, since I display only words, and even 'w' is obsolete in this case.

**Idea**

Allow to set a custom suffix instead of just abbreviated one.

**Comment by isaaclyman** [comment]
Apologies, but I find it hard to believe a "w" bothers anyone enough to justify complicating the plugin settings further. I'm gonna say no on this one, feel free to fork the plugin.

**Comment by gchegr** [comment]
Thanks! Testing it out.

When I checked 'Use advanced formatting', it used 'w' for the '[Note count] Custom suffix' for folders (while having 'notes' with the option disabled).
I replaced it with 'n' and it works.

For some reason the right alignment for folders is now off, and the data is put right after the folder name (even with the advanced formatting disabled):
![Obsidian_kQZSPJiW7z](https://github.com/isaaclyman/novel-word-count-obsidian/assets/81404846/5e8316e6-6a65-435c-a0df-5528168d825c)

As for the tip — I'd really love to :/ But Ko-fi is a Stripe-based service, and I can’t use it now, for known political reasons. PayPal is no longer supported here... Sorry :( I hope it changes someday.
